### PR TITLE
Prepare for deployment to development environment (closes #17).

### LINKED ISSUE #17: Deploy to a development environment.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @6a68

### CODE REVIEW

**Comment by chuckharmston** [comment]
One weird thing I'm working through: `python-memcached` has a py3 bug that has been fixed since the last release, but `setup.py`'s `install_requires` doesn't like the pip editable (`-e`) format.

**Comment by 6a68** [comment]
@chuckharmston Could you add some docs around how to deploy with elastic beanstalk?

**Comment by chuckharmston** [comment]
How's that? If those steps don't work for you, we should get on Vidyo and walk through it to make sure every necessary step is captured. Unfortunately it's not particularly easy for me to start fresh myself with a new user.

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ Universal Search recommendation server.
 
 [![Build Status](https://travis-ci.org/mozilla/universal-search-recommendation.svg?branch=master)](https://travis-ci.org/mozilla/universal-search-recommendation) [![Coverage Status](https://coveralls.io/repos/mozilla/universal-search-recommendation/badge.svg?branch=master&service=github)](https://coveralls.io/github/mozilla/universal-search-recommendation?branch=master) [![Requirements Status](https://requires.io/github/mozilla/universal-search-recommendation/requirements.svg?branch=master)](https://requires.io/github/mozilla/universal-search-recommendation/requirements/?branch=master)
 
+**Documentation**: [Local Development](#local-development) | [Methodology](#methodology) | [Deployment](docs/deploy.md)
```
Comment (chuckharmston) [inline README.md]: With #7 this will be better fleshed out.

**Comment by 6a68** [comment]
Dev docs look great, thanks so much for the help getting my EB configs in order. Merging.

**Pull request merged**